packages/react/react/src/hooks/useVisibilityEvent.en.md

```diff
@@ -1,3 +1,17 @@
 # useVisibilityEvent
 
 Call the callback function when the document's visibilitychange event occurs.
+
+## Example
+
+```tsx
+type VisibilityState = 'hidden' | 'visible';
```

Comment:
There is [the built-in type `DocumentVisibilityState` in Typescript](https://github.com/microsoft/TypeScript/blob/cbda2fcce95d7ac82841cee631f07b73be07c0f0/src/lib/dom.generated.d.ts#L27834).